src/multi.rs

```diff
@@ -169,8 +174,8 @@ impl MultiProgress {
 pub(crate) struct MultiState {
     /// The collection of states corresponding to progress bars
     /// the state is None for bars that have not yet been drawn or have been removed
-    draw_states: Vec<Option<DrawState>>,
-    /// Set of removed bars, should have corresponding `None` elements in the `draw_states` vector
+    elements: Vec<MultiStateElement>,
```

Comment:
Hmm, not so sure about the `element`/`Element` name. Maybe `members` or `bars`?

Also if there are a bunch of diff hunks that only change this name, might be a good idea to pull that change out into a separate commit.